src/c2ffi/package.lisp

```diff
@@ -0,0 +1,40 @@
+;;;; -*- Mode: lisp; indent-tabs-mode: nil -*-
+;;;
+;;; package.lisp --- Groveler DEFPACKAGE.
```

Comment:
can i just delete that line altogether? it's a perfect example of harmful documentation...